.github/workflows/cd_release.yml

```diff
@@ -47,12 +47,12 @@ on:
         type: string
         default: "3.9"
       package_dirs:
-        description: "A single or multi-line string of paths to Python package directories relative to the repository directory to have its `__version__` value updated. Example: `'src/my_package'`. **Important**: This is _required_ if 'python_package' is 'true', which is the default."
+        description: "A multi-line string of paths to Python package directories relative to the repository directory to have its `__version__` value updated. Example: `'src/my_package'`. **Important**: This is _required_ if 'python_package' is 'true', which is the default."
         required: false
         type: string
         default: ""
       version_update_changes:
-        description: "A single or multi-line string of changes to be implemented in the repository files upon updating the version. The string should be made up of three parts: 'file path', 'pattern', and 'replacement string'. These are separated by the 'version_update_changes_separator' value. The 'file path' must _always_ either be relative to the repository root directory or absolute. The 'pattern' should be given as a 'raw' Python string."
```

Comment:
From the code below it looks like single line is also accepted?

Reply:
For all of these descriptions, I have removed "single" if the parameter input is not carefully parsed.
Essentially, it means: It can definitely still be a single line, but it cannot be a multi-input/multi-value input within a single line.

So some input examples:

**Accepted** input styles:

```yaml
# Two separate version update changes:
version_update_changes: |
  "file/path,pattern,replacement string"
  "another/file/path,pattern,replacement string"

# A single version update change
version_update_changes: |
  "file/path,pattern,replacement string"

# A single version update change, different formatting for input
version_update_changes: "file/path,pattern,replacement string"
```

**Disallowed** input styles:
```yaml
# Two separate version update changes:
version_update_changes: "file/path,pattern,replacement string another/file/path,pattern,replacement string"
```

Whereas, other input parameters **does** multiple values within a single line, where the multiple values are then separated by a space.

Reply:
Ah, these examples are vert nice formatted like this. I could not find them in the documentation. Can you point me to where it ends up in the documentation or add them?

Reply:
I'll add them :)